### PR TITLE
[NotSoBot] fix `[p]gascii` command

### PR DESCRIPTION
**Pull request by owocado**
Hi,

As discussed, here is my fix for `[p]gascii` command.

There were few other ways to do this to achieve the same result but I decided to implement it this way. Please review my contribution and please let me know if there any any changes/improvements required.

I have tested my changes, and it seems to have worked successfully.

### CODE REVIEW

File: notsobot/notsobot.py [hunk 1]
```diff
@@ -718,13 +719,14 @@ def do_gascii(self, b):
...
+                img_list.append(img)
                 count += 1
-            temp = BytesIO()
-            new_im.save(
-                temp, format="GIF", save_all=True, append_images=img_list, duration=0, loop=0
-            )
+                temp = BytesIO()
+                img.save(
+                    temp, format="GIF", save_all=True, append_images=img_list, duration=0, loop=0
+                )
```
Comment (TrustyJAID) [inline notsobot/notsobot.py]: Saving inside this for-loop is a very taxing operation are you sure this is how it is intended to work?

Comment (owocado) [reply notsobot/notsobot.py]: Ah yes, you're absolutely right. I have moved that outside of for loop. Tested my changes and it works.

**Review by TrustyJAID — APPROVED** [review]
LGTM, thanks for the fix!

**Comment by owocado** [comment]
Thank you very much for accepting my contribution. 😅

**Pull request merged**